pages/app-layout/sidecar-demo.page.tsx

```diff
@@ -37,6 +41,112 @@ type DemoContext = React.Context<
   }>
 >;
 
+const AIDrawer = () => {
+  return (
+    <Box padding="m">
+      <Box variant="h2" padding={{ bottom: 'm' }}>
+        Chat demo
+      </Box>
+      <AutoIncrementCounter />
+      {new Array(100).fill(null).map((_, index) => (
+        <div key={index}>Tela content</div>
+      ))}
+    </Box>
+  );
+};
+
+awsuiPlugins.appLayout.registerDrawer({
```

Comment:
This seems to be a copy paste from another page. All same comments apply.

Plus a suggestion to extract this in a shared file is more important now